Makefile

```diff
@@ -143,12 +143,10 @@ test-npm: $(NODE_EXE)
 	cd deps/npm ; npm_config_cache="$(shell pwd)/npm-cache" \
 	     npm_config_prefix="$(shell pwd)/npm-prefix" \
 	     npm_config_tmp="$(shell pwd)/npm-tmp" \
-	     ../../$(NODE_EXE) cli.js install --ignore-scripts
-	cd deps/npm ; npm_config_cache="$(shell pwd)/npm-cache" \
-	     npm_config_prefix="$(shell pwd)/npm-prefix" \
-	     npm_config_tmp="$(shell pwd)/npm-tmp" \
+	     ../../$(NODE_EXE) cli.js install --ignore-scripts && \
```

Comment:
Sounds more like a broken .gitignore file than a broken Makefile to me ;-)

But I see your point. Tests that frustrate the people who are encouraged to run them are self defeating.